### PR TITLE
Release notes for 5.6.43-84.3

### PR DESCRIPTION
**Pull request by borys-belinsky**

### CODE REVIEW

File: doc/source/release-notes/Percona-Server-5.6.43-84.3.rst [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+all the bug fixes in it. |Percona Server| |release| is now the current GA
+release in the 5.6 series. All of Percona’s software is open-source and free.
+
+Bugs Fixed
+================================================================================
+
+- A sequence of LOCK TABLES FOR BACKUP and STOP SLAVE SQL_THREAD could cause
+  replication to be blocked and not possible to be restarted normally. Bug fixed
+  :psbug:`4758` (upstream :mysqlbug:`93649`).
+- In some cases, it was possible to inject SQL into the binary log due to
```
Comment (laurynas-biveinis) [inline doc/source/release-notes/Percona-Server-5.6.43-84.3.rst]: Other bugs. This is a fix we inherited by merging upstream release

File: doc/source/release-notes/Percona-Server-5.6.43-84.3.rst [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  non-quoting in ROLLBACK TO SAVEPOINT. Bug fixed :psbug:`4791` (upstream
+  :mysqlbug:`92227`).
+- **http** with **https** in http://bugs.percona.com in server crash
```
Comment (laurynas-biveinis) [inline doc/source/release-notes/Percona-Server-5.6.43-84.3.rst]: Missing verb

File: doc/source/release-notes/Percona-Server-5.6.43-84.3.rst [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+  messages. Bug fixed :psbug:`4855`.
+- Inconsistent ResultSet could be obtained for different execution plans. Bug
```
Comment (laurynas-biveinis) [inline doc/source/release-notes/Percona-Server-5.6.43-84.3.rst]: Please use standard MySQL terms, not "ResultSet". Description too vague, please check with @percona-ysorokin for a more exact description

File: doc/source/release-notes/Percona-Server-5.6.43-84.3.rst [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+  fixed :psbug:`4907` (upstream bug :mysqlbug:`92809`).
+- The audit logs could be corrupted due to an invalid size of the audit log file
```
Comment (laurynas-biveinis) [inline doc/source/release-notes/Percona-Server-5.6.43-84.3.rst]: Invalid size is not a cause of corruption, it is its outcome

File: doc/source/upstream-bug-fixes.rst [hunk 5]
```diff
@@ -4,6 +4,24 @@
...
 =============================================================
 
++-------------------------------------------------------------------------------------------------------------+
+|:Upstream bug: :mysqlbug:`93649` - STOP SLAVE SQL_THREAD deadlocks if done while holding LOCK INSTANCE   ... |
+|:JIRA bug: :psbug:`4758`                                                                                     |
+|:Upstream state: Open (checked on 2018-05-29)                                                                |
+|:Fix Released: :rn:`5.6.43-84.3`                                                                             |
+|:Upstream fix: N/A                                                                                           |
++-------------------------------------------------------------------------------------------------------------+
+|:Upstream bug: :mysqlbug:`92809` - STOP SLAVE SQL_THREAD deadlocks if done while holding LOCK INSTANCE   ... |
```
Comment (laurynas-biveinis) [inline doc/source/upstream-bug-fixes.rst]: Wrong title

File: doc/source/upstream-bug-fixes.rst [hunk 6]
```diff
@@ -4,6 +4,24 @@
...
+|:JIRA bug: :psbug:`4907`                                                                                     |
+|:Upstream state: Verified (checked on 2018-05-29)                                                            |
+|:Fix Released: :rn:`5.6.43-84.3`                                                                             |
+|:Upstream fix: N/A                                                                                           |
++-------------------------------------------------------------------------------------------------------------+
+|:Upstream bug: :mysqlbug:`92227` - /etc/percona-server.cnf (centos) and /etc/mysql/percona-server.cnf    ... |
```
Comment (laurynas-biveinis) [inline doc/source/upstream-bug-fixes.rst]:
```suggestion
|:SQL injection on slave due to non-quoting in binlogged ROLLBACK TO SAVEPOINT |
```

**Review by laurynas-biveinis — CHANGES_REQUESTED** [review]

File: doc/source/release-notes/Percona-Server-5.6.43-84.3.rst (hunk 1 above)
Comment (borys-belinsky) [reply doc/source/release-notes/Percona-Server-5.6.43-84.3.rst]: Ready for review

File: doc/source/release-notes/Percona-Server-5.6.43-84.3.rst [hunk 7]
```diff
@@ -0,0 +1,63 @@
...
+
+Bugs Fixed
+================================================================================
+
+- A sequence of LOCK TABLES FOR BACKUP and STOP SLAVE SQL_THREAD could cause
+  replication to be blocked and not possible to be restarted normally. Bug fixed
+  :psbug:`4758` (upstream :mysqlbug:`93649`).
+- **http** was replaced with **https** in http://bugs.percona.com in server
+  crash messages. Bug fixed :psbug:`4855`.
+- Fixed a bug in materialization-scan that allowed inner tables of different
```
Comment (laurynas-biveinis) [inline doc/source/release-notes/Percona-Server-5.6.43-84.3.rst]: Please add back the effect for the end user: wrong query result

**Review by laurynas-biveinis — CHANGES_REQUESTED** [review]

File: doc/source/release-notes/Percona-Server-5.6.43-84.3.rst (hunk 7 above)
Comment (borys-belinsky) [reply doc/source/release-notes/Percona-Server-5.6.43-84.3.rst]: Ready for review

**Review by laurynas-biveinis — APPROVED** [review]

**Pull request merged**